kp_pre_commit_hooks/gitops-values-validation.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python3
+
+import json
+import sys
+from itertools import chain
+from pathlib import Path
+
+import requests
+import urllib3
+from jsonschema import validate
+from jsonschema.exceptions import ValidationError
+from ruamel.yaml import YAML
+
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+SCHEMA_BASE_URL = "https://kp-helmchart-stable-shared-main.s3.eu-west-1.amazonaws.com/schema"
+GITOPS_DIR = Path("gitops")
+
+yaml = YAML()
+
+
+def download_json_schema_for_chart_version(version):
+    schema_url = f"{SCHEMA_BASE_URL}/v{version}/schema-platform-managed-chart-strict.json"
+    response = requests.get(schema_url, timeout=10, verify=False)
+
+    if response.status_code != 200:
+        print(f"Error fetching schema url {schema_url}. HTTP Status Code: {response.status_code}")
+        return None
+
+    try:
+        schema_json = response.json()
+        return schema_json
+    except json.JSONDecodeError:
+        print(f"Error decoding JSON. Response content:\n{response.text}")
```

Comment:
Here do you prefer to not fail to not block any deployment in case we make a mistake in a json schema ?